eslint.config.js

```diff
@@ -36,5 +36,14 @@ export default [
       ],
     },
   },
+  // https://stackoverflow.com/questions/48584556/eslint-chrome-is-not-defined-no-undef
```

Comment:
[no-undef](https://eslint.org/docs/latest/rules/no-undef)
eslint공식문서에 있는 규칙이였네요.

```suggestion
  /*global chrome*/
```
위와 같이 에러가 난 파일에 작성하면 eslint에게 chrome이 전역적으로 선언되었다고 알려주는 것이라고 합니다.

Reply:
오 해당 구문은 처음 보는데 덕분에 접할 수 있었어요 종석님 감사합니다. 

찾아보니 /*global chrome*/은 각각 해당 전역(chrome)에 접근하는 모든 파일 상단에 남겨야 하는 방법인 것 같아요. 현재 chrome에 접근하는 파일 갯수가 미래에 몇 개가 될지 짐작하기 어려운 것 같습니다. 
다만, 제공해주신 공식문서 중 링크를 타고타고 들어가다보니 [predefined global variables](https://eslint.org/docs/latest/use/configure/language-options#predefined-global-variables) 라는 좋은 카테고리를 발견해서 해당 주소로 주석을 변경하도록 하겠습니다!

Reply:
궁금할 때 PR로 다시 올 수 있으니 해당 주석은 삭제하도록 하겠습니다